docs/docs/tutorial/part-5/index.mdx

```diff
@@ -0,0 +1,530 @@
+
+
+### Task: Install and configure the MDX transformer plugin (and dependencies)
+
+The `gatsby-plugin-mdx` package requires a few additional dependencies to run: `@mdx-js/mdx` and `@mdx-js/react`. These are both packages that DO SOMETHING.
+
+Run the command below in the terminal to install `gatsby-plugin-mdx` and its dependencies.
+
+```shell
+npm install gatsby-plugin-mdx @mdx-js/mdx @mdx-js/react
+```
+
+```js:title=gatsby-config.js
+module.exports = {
+  siteMetadata: {
+    title: "My Super Cool Blog",
+  },
+  plugins: [
+    "gatsby-plugin-gatsby-cloud",
+    "gatsby-plugin-image",
+    "gatsby-plugin-sharp",
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `blog`,
+        path: `${__dirname}/blog/`,
+      },
+    },
+    "gatsby-plugin-mdx", // highlight-line
+  ],
+};
+```
+
+// WORKSHOP
+
+The `gatsby-plugin-mdx` plugin provides some new tools for you to use in your site:
+
+* The `allMdx` and `mdx` fields (for your GraphQL queries)
+* An `MDXRenderer` component (for processing and displaying MDX content)
+
+You'll take a closer look at both of these pieces in the next few sections.
+
+// END WORKSHOP
+
+<Announcement style={{marginBottom: "1.5rem"}}>
+
+**Tip:** There are a variety of [remark](https://remark.js.org/) plugins that you can use to add extra features to your Markdown. You can configure them using the [`gatsbyRemarkPlugins`](/plugins/gatsby-plugin-mdx/#gatsby-remark-plugins) option when you configure `gatsby-plugin-mdx` in your `gatsby-config.js` file.
+
+Here are some popular remark plugins:
+
+* [`gatsby-remark-images`](/plugins/gatsby-remark-images/): Use this if you want to use the Markdown image syntax (which looks like this: `![alt](image url)`).
+    * To use this plugin, you'll also need `gatsby-plugin-sharp`, which you installed already in Part 3.
+* [`gatsby-remark-prismjs`](/plugins/gatsby-remark-prismjs/): Add syntax highlighting to your code blocks.
+* [`gatsby-remark-autolink-headers`](/plugins/gatsby-remark-autolink-headers/): Automatically create links for all the headers in your Markdown content.
+
+Try searching for `gatsby-remark-` in the [Gatsby Plugin Library](/plugins/) for a full list.
+
+</Announcement>
+
+### Task: query
+
+In GraphiQL, there are now two new fields available for you to use in your queries: `allMdx` and `mdx`. In this part of the Tutorial, you'll use `allMdx` to add the contents of each blog post to your Blog page. (You'll use `mdx` later on, in Part 6.)
+
+<Collapsible
+  summary="allMdx field in GraphiQL"
+>
+
+<Announcement style={{marginBottom: "1.5rem"}}>
+
+You can see fields from the original node in the `parent` field. TRY IT IN GRAPHIQL. THE ORIGINAL NODES WILL STILL BE THERE.
+
+- SHOULD WE TEACH THEM ABOUT THE PARENT FIELD? possibly use the parent field to get access to the "changeTime" field, so they can add a "Updated: XXXXX" date to their post? (Possibly have to teach them about GraphQL fragments then...) (only as a callout.)
```

Comment:
Maybe as a bonus or extra lesson for people who want to learn this? I'd definitely would flag this under "definitely helpful in the long run as at some point you'll need to interact with this / need this functionality" and already teaching it here would be super cool.

Reply:
Nice! I added it as a "Pro Tip" callout box, so hopefully folks who get confused by it will just skip over it.